test/module/shared_model/backend_proto/shared_proto_tx_response_test.cpp

```diff
@@ -46,3 +48,32 @@ TEST(ProtoTxResponse, TxResponseLoad) {
               shared_model::crypto::Hash(hash));
   });
 }
+
+/**
+ * @given TransactionResponse that previously had lazy fields
```

Comment:
"uninitialized lazy fields"

Reply:
The point is not on their initialisation state. The point is on presence of lazy fields.